doc/getting_started.md

```diff
@@ -0,0 +1,556 @@
+
+This function is used to retrieve a wallet from the given mnemonics and password.
+It will work for all yoroi, daedalus and other wallets as it will try every kind of wallet anyway
+
+Can also be used to recover a wallet even after you have transferred all the funds to the new format (see the [wallet_convert](#wallet_convert) function), just keep in mind that it will reset the wallet internal state.
+
+---
+
+### wallet_id
+
+- **input**:
+  - `wallet` - retireved from mnemonics ([wallet_recover](#wallet_recover))
+- **output**:
+  - `walled_id` - (aka accountID) used against the blockchain/explorer to retrieve the state of the wallet
+  - `error` - returned in case the wallet is not yet recovered
+
+This is the wallet identifier (aka accountID) that is to be used against the blockchain/explorer
+to retrieve the state of the wallet (counter, total value, ...)
+
+#### How to query the network for the wallet_id information
+
+API calls used:
+>Get: /v0/**account/{wallet_id}**
+>
+> - **wallet_id** value has to be a string in hexadecimal encoding
+
+The network can be queried by using the Rest API interface.
+
+> - Node rest api interface is at **http://127.0.0.1:8001/api**
+> - wallet_id is **daa45bd19e1f6d2fce25542c2bda11076d4b359d3b5a6dd5b9588e2f34569503**
+
+```json
+Request {
+    method: GET,
+    url: "http://127.0.0.1:8001/api/v0/account/daa45bd19e1f6d2fce25542c2bda11076d4b359d3b5a6dd5b9588e2f34569503",
+    headers: {},
+}
+```
+
+When issued the first time, the response body of that query will be empty and you will get status **404 Not Found**
+since the account doesn't exist yet on the chain:
+
+```json
+Response {
+    url: "http://127.0.0.1:8001/api/v0/account/daa45bd19e1f6d2fce25542c2bda11076d4b359d3b5a6dd5b9588e2f34569503",
+    status: 404,
+    headers: {
+        "content-length": "0",
+        "date": "Thu, 14 May 2020 18:51:12 GMT",
+    },
+}
+```
+
+When issued after the convert process is completed:
+
+```json
+Response {
+    url: "http://127.0.0.1:8001/api/v0/account/daa45bd19e1f6d2fce25542c2bda11076d4b359d3b5a6dd5b9588e2f34569503",
+    status: 200,
+    headers: {
+        "content-type": "application/json",
+        "content-length": "91",
+        "date": "Thu, 14 May 2020 19:24:54 GMT",
+    },
+}
+Response body:
+{
+  "counter": 0,
+  "delegation": {
+    "pools": []
+  },
+  "last_rewards": {
+    "epoch": 0,
+    "reward": 0
+  },
+  "value": 20089
+}
+```
+
+The interesting parts here are the **counter** and **value** that will be used from [wallet_set_state](#wallet_set_state)
+and from other interactions with the node (vote casting, ...)
+
+---
+
+### wallet_retrieve_funds
+
+- **input**:
+  - `wallet` - recovered from mnemonics ([wallet_recover](#wallet_recover))
+  - `block0` - content that has to be retrieved from the blockchain network using the rest api
+- **output**:
+  - `wallet` - with initial funds retrieved from block0 (not yet converted)
```

Comment:
this is in a conceptual level, right? Because that's actually a side-effect in the implementation.

Reply:
Yes, input/output are conceptual. Tried to keep it that way just to keep it general.